src/RtcTemperature.h

```diff
@@ -7,32 +7,54 @@
 class RtcTemperature
 {
 public:
-    RtcTemperature(int8_t degrees, uint8_t fraction) :
-        integerDegrees(degrees),
-        decimalFraction(fraction)
+    RtcTemperature( int8_t msdegC, uint8_t lsdegC ) :
+        mstemp( msdegC ),
+        lstemp( lsdegC & 0xC0 )  // Bind to RTC resolution
     {
     }
 
+    // Scaled integer temperature (x 256)
+    int16_t AsScaledDegrees()
```

Comment:
I don't like the name of this just due to the fact that its basically the raw "8.8 fixed point" math item.  
"AsFixedPointDegrees" would be my suggestion.